<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>125</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Notepy</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow {background: white; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="editor">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="file">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_new_window"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_page_settings"/>
    <addaction name="action_print"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="edit">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="separator"/>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="action_delete"/>
    <addaction name="separator"/>
    <addaction name="action_search"/>
    <addaction name="action_find_next"/>
    <addaction name="action_find_previous"/>
    <addaction name="action_replace"/>
    <addaction name="action_goto"/>
    <addaction name="separator"/>
    <addaction name="action_select_all"/>
    <addaction name="action_datetime"/>
    <addaction name="separator"/>
    <addaction name="action_font"/>
   </widget>
   <widget class="QMenu" name="view">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Ver</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="tearOffEnabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="action_zoom_in"/>
     <addaction name="action_zoom_out"/>
     <addaction name="action_restore_zoom"/>
    </widget>
    <addaction name="menuZoom"/>
    <addaction name="action_status_bar"/>
    <addaction name="action_word_wrap"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="action_settings"/>
    <addaction name="action_about"/>
    <addaction name="action_about_qt"/>
   </widget>
   <addaction name="file"/>
   <addaction name="edit"/>
   <addaction name="view"/>
   <addaction name="help"/>
  </widget>
  <widget class="QStatusBar" name="status_bar">
   <property name="styleSheet">
    <string notr="true">QStatusBar { background: #cccccc; }</string>
   </property>
  </widget>
  <action name="action_new">
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_new_window">
   <property name="text">
    <string>Nueva ventana</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>Guardar como</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_page_settings">
   <property name="text">
    <string>Configurar página</string>
   </property>
  </action>
  <action name="action_print">
   <property name="text">
    <string>Imprimir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>Deshacer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="text">
    <string>Cortar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="text">
    <string>Copiar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="text">
    <string>Pegar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>Eliminar</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_search">
   <property name="text">
    <string>Buscar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_find_next">
   <property name="text">
    <string>Buscar siguiente</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_find_previous">
   <property name="text">
    <string>Buscar anterior</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="action_replace">
   <property name="text">
    <string>Reemplazar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_goto">
   <property name="text">
    <string>Ir a</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_select_all">
   <property name="text">
    <string>Seleccionar todo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_datetime">
   <property name="text">
    <string>Fecha y hora</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_font">
   <property name="text">
    <string>Fuente</string>
   </property>
  </action>
  <action name="action_zoom_in">
   <property name="text">
    <string>Acercar</string>
   </property>
  </action>
  <action name="action_zoom_out">
   <property name="text">
    <string>Alejar</string>
   </property>
  </action>
  <action name="action_restore_zoom">
   <property name="text">
    <string>Restaurar zoom predeterminado</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="action_status_bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Barra de estado</string>
   </property>
  </action>
  <action name="action_word_wrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ajuste de línea</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>Configuración</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="text">
    <string>Acerca de Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
